2996_reflection/reflection.md

```diff
@@ -2585,6 +2585,14 @@ In fact, that is recommended practice.
 ```c++
 namespace std::meta {
   struct data_member_options_t {
+    struct name_type {
+      template <typename T> requires constructible_from<u8string, T>
+        consteval name_type(T &&);
+
+      template <typename T> requires constructible_from<string, T>
+        consteval name_type(T &&);
+    };
+
     optional<string_view> name;
```

Comment:
This should be `optional<name_type>` presumably?

Reply:
Oops,  yep. Thanks.